### ISSUE TITLE
realtime effect dialogs: playback cannot be started or stopped using the keyboard within the dialog

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#3483]
### Bug description

realtime effect dialogs: playback cannot be started or stopped using the keyboard within the dialog - you have to make the main window the focus to be able to do that.

### Steps to reproduce

1. Open a realtime effect.
2.  Try to start/stop playback without leaving the dialog.

### Expected behavior

You can.

### Actual behavior

You can't.

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

### Additional context

This is a regression.

**Comment by LWinterberg** [comment]
I would expect for `Space` to not toggle the power button but always play.

**Comment by DavidBailes** [comment]
> I would expect for `Space` to not toggle the power button but always play.

Are you suggesting this behaviour for the power button, but not for any other controls in the dialog?

**Comment by LWinterberg** [comment]
no, for everything - I'd expect it to always play no matter what's selected

**Comment by dozzzzer** [comment]
Verified on macOS and Windows.
